### ISSUE TITLE
Template not found: nunjucks recipe

### ISSUE THREAD

**Issue by CesarDenis** [yeoman/generator-webapp#449]
I followed the recipe [nunjucks](https://github.com/yeoman/generator-gulp-webapp/blob/master/docs/recipes/nunjucks.md) but when I run the task serve the following error happens:

```
events.js:154
      throw er; // Unhandled 'error' event
      ^
Template render error: (unknown path)
  Error: template not found: layouts/default.html
    at Object.exports.prettifyError (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/lib.js:34:15)
    at /Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/environment.js:485:31
    at eval (eval at <anonymous> (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/environment.js:564:24), <anonymous>:9:11)
    at createTemplate (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/environment.js:212:25)
    at next (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/lib.js:210:13)
    at handle (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/environment.js:252:25)
    at /Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/environment.js:263:21
    at next (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/lib.js:207:13)
    at Object.exports.asyncIter (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/lib.js:214:5)
    at [object Object].Obj.extend.getTemplate (/Users/macweb01/Sites/ccr/ras/node_modules/nunjucks/src/environment.js:242:17)
```

The [repo](https://bitbucket.org/adesignet/ras-ccr-html-site/src/bd0545bdc289ad7747e0569fc6559c0a211b13f2?at=master) of my project.

Can someone help me?

Best regards

**Comment by CesarDenis** [comment]
I was able to ride changing the path the layout.

In `app/index.html`:

```diff
-{% extends "layouts/default.html" %}
+{% extends "app/layouts/default.html" %}

{% block content %}
  <div class="hero-unit">
    <h1>'Allo, 'Allo!</h1>
    <p>You now have</p>
    <ul>
      <li>HTML5 Boilerplate</li>
      <li>Sass</li>
      <li>Modernizr</li>
    </ul>
  </div>
{% endblock %}
```

This correct or is the wrong setting?

**Comment by silvenon** [comment]
We should really specify versions that our recipes work in :sweat_smile: There was a breaking change in gulp-nunjucks-render v2, so try using `path` in the configuration instead of using `$.nunjucksRender.configure`, like [this](https://github.com/carlosl/gulp-nunjucks-render#example). Let me know if that worked.

**Comment by CesarDenis** [comment]
It worked ... :smiley:
My task view:

```
gulp.task('views', () => {
  return gulp.src('app/*.html')
    .pipe($.nunjucksRender({
      path: ['app/']
    }))
    .pipe(gulp.dest('.tmp'))
});
```

Tks @silvenon

**Comment by silvenon** [comment]
You can make a PR to fix the recipe if you want to. Otherwise I'll do it.

**Comment by CesarDenis** [comment]
I do, thank you. :smiley:

**Comment by joelbschwartz** [comment]
I also addressed this with PR #456 - but that includes some other changes as well...! :)
